### PR TITLE
fix: focusAsync still works even if past animationFrame fails to run

### PR DESCRIPTION
**Pull request by smhigley**
This came up while helping a partner debug why focus wasn't entering `ContextualMenu` after closing & re-opening an email window. It turns out that the current implementation of `focusAsync` fails to call focus if a previous call of `focusAsync` set `targetToFocusOnNextRepaint` but didn't run `win.requestAnimationFrame` (e.g. because the window closed).

By saving the `requestAnimationFrame` and cancelling prior ids on subsequent same-animation-frame calls, we can ensure that a past not-run `focusAsync` does not prevent future focus calls.

### CODE REVIEW

File: change/@fluentui-utilities-2c559317-d528-4f32-8c1a-0ad2f000dc12.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (fabricteam) [inline change/@fluentui-utilities-2c559317-d528-4f32-8c1a-0ad2f000dc12.json:1]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=29322&commitId=a6cd7520c0cdb1b2ffd7ca4722ddb254dc96dad6&env=prod&clientType=FLUENTUI)


  <details><summary><b>TeachingBubble</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|TeachingBubble.WideIllustration - RTL.default.chromium.png|395| Changed|
|TeachingBubble.SmallHeadline.default.chromium.png|395| Changed|
|TeachingBubble.WideIllustration.default.chromium.png|395| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/8dd29fff580dac6fe127b2f23ebd4bd6ed1f46f9'>8dd29fff580dac6fe127b2f23ebd4bd6ed1f46f9</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/BaseButton-mount_base.html">637</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/BaseButton-mount.html">674</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/Breadcrumb-mount_base.html">1729</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/Breadcrumb-mount.html">1763</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/Checkbox-mount_base.html">1713</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/Checkbox-mount.html">1693</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/CheckboxBase-mount_base.html">1511</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/CheckboxBase-mount.html">1469</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/ChoiceGroup-mount_base.html">2938</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/ChoiceGroup-mount.html">2931</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/ComboBox-mount_base.html">650</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/ComboBox-mount.html">666</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/CommandBar-mount_base.html">6430</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/CommandBar-mount.html">6517</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/ContextualMenu-mount_base.html">16159</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/ContextualMenu-mount.html">16526</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/DefaultButton-mount_base.html">738</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/DefaultButton-mount.html">755</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/DetailsRow-mount_base.html">2231</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/DetailsRow-mount.html">2207</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/DetailsRowFast-mount_base.html">2173</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/DetailsRowFast-mount.html">2250</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/DetailsRowNoStyles-mount_base.html">2065</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/DetailsRowNoStyles-mount.html">2049</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/Dialog-mount_base.html">2657</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/Dialog-mount.html">2766</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/DocumentCardTitle-mount_base.html">232</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/DocumentCardTitle-mount.html">246</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/Dropdown-mount_base.html">2005</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/Dropdown-mount.html">1987</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/FocusTrapZone-mount_base.html">1092</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/FocusTrapZone-mount.html">1161</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/FocusZone-mount_base.html">1087</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/FocusZone-mount.html">1078</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/GroupedList-mount_base.html">42300</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/GroupedList-mount.html">42398</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/GroupedList-virtual-rerender_base.html">20149</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/GroupedList-virtual-rerender.html">20354</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/GroupedList-virtual-rerender-with-unmount_base.html">53093</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/GroupedList-virtual-rerender-with-unmount.html">52000</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/GroupedListV2-mount_base.html">232</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/GroupedListV2-mount.html">231</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/GroupedListV2-virtual-rerender_base.html">215</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/GroupedListV2-virtual-rerender.html">220</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/GroupedListV2-virtual-rerender-with-unmount_base.html">238</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/GroupedListV2-virtual-rerender-with-unmount.html">241</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/IconButton-mount_base.html">1082</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/IconButton-mount.html">1062</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/Label-mount_base.html">340</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/Label-mount.html">344</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/Layer-mount_base.html">2796</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/Layer-mount.html">2770</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/Link-mount_base.html">408</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/Link-mount.html">392</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/MenuButton-mount_base.html">938</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/MenuButton-mount.html">953</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/MessageBar-mount_base.html">21647</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/MessageBar-mount.html">21804</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/Nav-mount_base.html">1951</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/Nav-mount.html">1956</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/OverflowSet-mount_base.html">788</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/OverflowSet-mount.html">787</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/Panel-mount_base.html">2073</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/Panel-mount.html">1782</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/Persona-mount_base.html">732</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/Persona-mount.html">735</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/Pivot-mount_base.html">852</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/Pivot-mount.html">847</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/PrimaryButton-mount_base.html">849</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/PrimaryButton-mount.html">856</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/Rating-mount_base.html">4615</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/Rating-mount.html">4644</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/SearchBox-mount_base.html">899</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/SearchBox-mount.html">914</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/Shimmer-mount_base.html">1897</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/Shimmer-mount.html">1900</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/Slider-mount_base.html">1360</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/Slider-mount.html">1321</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/SpinButton-mount_base.html">2950</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/SpinButton-mount.html">2838</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/Spinner-mount_base.html">395</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/Spinner-mount.html">390</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/SplitButton-mount_base.html">1824</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/SplitButton-mount.html">1826</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/Stack-mount_base.html">417</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/Stack-mount.html">416</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/StackWithIntrinsicChildren-mount_base.html">842</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/StackWithIntrinsicChildren-mount.html">877</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/StackWithTextChildren-mount_base.html">2644</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/StackWithTextChildren-mount.html">2650</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/SwatchColorPicker-mount_base.html">6164</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/SwatchColorPicker-mount.html">6234</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/TagPicker-mount_base.html">1512</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/TagPicker-mount.html">1541</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/Text-mount_base.html">365</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/Text-mount.html">391</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/TextField-mount_base.html">942</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/TextField-mount.html">957</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/ThemeProvider-mount_base.html">840</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/ThemeProvider-mount.html">853</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/ThemeProvider-virtual-rerender_base.html">596</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/ThemeProvider-virtual-rerender.html">583</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/ThemeProvider-virtual-rerender-with-unmount_base.html">1283</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/ThemeProvider-virtual-rerender-with-unmount.html">1284</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/Toggle-mount_base.html">619</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/Toggle-mount.html">616</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>buttonNative</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/buttonNative-mount_base.html">191</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29322/perf-test/buttonNative-mount.html">199</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9423"/>

**Review by spmonahan — APPROVED** [review]

File: change/@fluentui-utilities-2c559317-d528-4f32-8c1a-0ad2f000dc12.json (hunk 1 above)
Comment (smhigley) [reply change/@fluentui-utilities-2c559317-d528-4f32-8c1a-0ad2f000dc12.json:1]: Looks like the change fixed a prior TeachingBubble focus issue 😅

**Pull request merged**